### PR TITLE
Updated Bundle

### PR DESCRIPTION
**Pull request by KaiyiLiu1234**
Bundle has been updated with most up to date docker image. Once this PR is approved, the kepler operator v0.4 should be completed. Note that the Operator sdk version in the bundle went from 1.25 to 1.23 after updating the bundle. Will this be an issue?

### CODE REVIEW

**Comment by KaiyiLiu1234** [comment]
I just copy and pasted the icon into clusterserviceversion.yaml in bundle and config manifest. Not sure if there was a programatic way of doing it.

**Comment by KaiyiLiu1234** [comment]
@husky-parul I noticed that github actions are still failing because of the username password missing. Is there anything that can be done to resolve that?

**Comment by husky-parul** [comment]
image

- Add repository name
- Add CREATED AT
- Add channel as `v1alpha1`
- Add quay.io image link under container image
- Add https://github.com/sustainable-computing-io/kepler-operator in created at
- Need Introduction, Documentation, Contributing and License information. Follow sample csv here https://operatorhub.io/operator/openshift-qiskit-operator

You can preivew the csv here https://operatorhub.io/preview

**Review by husky-parul — CHANGES_REQUESTED** [review]
I have added some comments.

**Comment by husky-parul** [comment]
> @husky-parul I noticed that github actions are still failing because of the username password missing. Is there anything that can be done to resolve that?

The checks have passed. Once the PR is merged the workflow will build and push the image.

File: config/manifests/bases/kepler-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -71,4 +350,4 @@ spec:
   provider:
     name: https://sustainable-computing.io/
```
Comment (husky-parul) [inline config/manifests/bases/kepler-operator.clusterserviceversion.yaml:351]: Name is string. Use "Kepler Operator Contributors" That is why the link is not rendering.

**Review by husky-parul — APPROVED** [review]
LGTM

**Pull request merged**